cellmlmanip/model.py

```diff
@@ -567,6 +568,65 @@ def get_symbol_by_cmeta_id(self, cmeta_id):
 
         raise KeyError('No variable with cmeta id "%s" found.' % str(cmeta_id))
 
+    def get_symbol_by_ontology_term(self, namespace, local_name):
+        """
+        Searches the RDF graph for a variable annotated with the given
+        ``namespace:local_name`` and returns its symbol.
```

Comment:
`namespace` here isn't a prefix, so this doc is potentially confusing. I'd call the parameter `namespace_uri` and describe this as `{namespace_uri}local_name` which is a standard representation; similarly in the next para and error messages below.

Reply:
Done!